lib/beaker/dsl/helpers.rb

```diff
@@ -1355,6 +1355,26 @@ def curl_on(host, cmd, opts = {}, &block)
           on host, "curl %s" % cmd, opts, &block
         end
       end
+
+      # Write hiera config file
+      #
+      # @param[Host, Array<Host>, String, Symbol] host    One or more hosts to act upon,
+      #                           or a role (String or Symbol) that identifies one or more hosts.
+      # @param[Array] One or more hierarchy paths
+      # @!macro common_opts
+      def write_hiera_config(host, hierarchy, opts = {})
```

Comment:
The beaker naming convention is that methods that don't end in _on run against the default node.  See 'run_script' vs. 'run_script_on'.  This should run against the default node and write_hiera_config_on should run against a provided array of nodes.